cordapp-cpk/src/main/kotlin/net/corda/plugins/cpk/DependencyCalculator.kt

```diff
@@ -28,6 +29,17 @@ open class DependencyCalculator @Inject constructor(objects: ObjectFactory) : De
         private const val NON_CORDA = false
         private const val CORDA = true
 
+        private val HARDCODED_EXCLUDES: Set<Pair<String, String>> = unmodifiableSet(setOf(
```

Comment:
why does it need to be wrapped in an `unmodifiableSet` ?

Reply:
Although a Kotlin `Set` cannot be modified, it is still a modifiable Java set underneath. I just want to make it _absolutely clear_ that this set is a constant.